ui/narrative/methods/view_proteinstructures/display.yaml

```diff
@@ -0,0 +1,47 @@
+
+#
+# Define basic display information
+#
+name     : View ProteinStructures
+subtitle : |
+    View and explore a ProteinStructures object in your workspace.
+tooltip  : |
+    View and explore a ProteinStructures object in your workspace.
+
+screenshots :
+    []
+
+#
+# Define the set of other narrative methods that should be suggested to the user.
+#
+suggestions :
+    apps:
+        related :
+            []
+        next :
+            []
+    methods:
+        related :
+            []
+        next :
+            []
+
+#
+# Configure the display and description of the parameters
+#
+parameters :
+    param0 :
```

Comment:
There is no `param0` in the spec file. This should be `proteinstructures`.

Reply:
Yes.